praw/util/__init__.py

```diff
@@ -1,3 +1,31 @@
 """Package imports for utilities."""
 
-__all__ = ("cache",)
+import re
+
+__all__ = ("cache", "_camel_to_snake_old", "camel_to_snake", "snake_case_keys")
+
+_re_camel_to_snake = re.compile(r"([a-z0-9](?=[A-Z])|[A-Z](?=[A-Z][a-z]))")
+
+
+def _camel_to_snake_old(name):
```

Comment:
Do we need the old method?

Reply:
Nope. It’s just there for the tests to prove that it’s pretty much equivalent to the new function.

The new regex behaves better on input that already contains underscores.

I will remove the old function now.